portfolio/src/main/webapp/script.js

```diff
@@ -174,17 +180,37 @@ function submitForm(form) {
     },
     body: JSON.stringify(formData),
   }).then(response => {
-    loadComments();
-    document.getElementById("newcommentform").reset();
+      loadComments();
+      document.getElementById("newcommentform").reset();
   });
 }
 
+/**
+ * Submits a post request to /delete-data to delete all comments,
+ * then reloads the comments section
+ */ 
 function clearComments() {
   fetch("/delete-data", {
     method: 'POST',
     headers: {
-      'Content-Type': 'application/json',
+      'Content-Type' : 'application/json',
```

Comment:
nit: I'm not sure if there is explicit style guides about this but object-like data should generally have a space following but not preceding a ":"